css/logo-nav.css

```diff
@@ -628,70 +627,56 @@ a {
     margin-bottom: 10px;
     cursor: default;
 } 
+
 .clicky:hover {
     color: #E46E2E
 }
-.institutional .items {
+
+.items {
```

Comment:
I merged the `institutional items` and the `donate-something` you where acchieving the same result with slightly different css. Usually these are [bootstrap wells](http://getbootstrap.com/components/#wells)